config/routes.rb

```diff
@@ -11,7 +11,7 @@
   get "/about",                    to: "public_pages#about",               as: "about_public"
   get "/announcements_list",       to: "public_pages#announcements",       as: "announcements_public"
   get "/community_resources_list", to: "public_pages#community_resources", as: "community_resources_public"
-  get "/contributions",            to: "public_pages#contributions",       as: "contributions_public" # TODO remove this
+  # get "/contributions",            to: "public_pages#contributions",       as: "contributions_public" # TODO remove this
```

Comment:
@maebeale I commented this out so that everything would work. Do you have a preference for having a public contributions path with one name, and a logged-in-only one with another name? 

It took me more time than I'd like to admit to figure out some tests were failing because two routes were listed for the "/contributions" path. I sure would like to see a louder warning from Rails when that happens.

Reply:
Hey! hmmm. I was thinking we'd move all of these routes to the ContributionsController, tho that doesn't answer re different endpoints for logged in vs not. 

My vote would be one endpoint, but I definitely see the value of having two. My worry is that we would start to implement something we want on both all the time, and get too divergent. My leaning is that we have one for a good while, until we see reason to have two. But, open to whatever y'all think.

Reply:
Cool, then I'll delete the commented out line for now, and then merge this